### ISSUE TITLE
Formatter for transitions as buttons

### ISSUE THREAD

**Issue by weitzman** [GBirch/massforms8#4]
https://www.drupal.org/node/2757317

**Comment by GBirch** [comment]
Agreed preliminary approach:  add possible end states as options on the standard drop-down submit button.  State machine guards will supply validation and error messages if conditions to transition are not met.
